src/rules/at-rule-name-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,460 @@
+  }, {
+    code: "@media\n(min-width: 700px)\r\nand (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px)\r\nand (orientation: landscape) { }",
+  }, {
+    code: "@supports\n(animation-name: test) { }",
+  }, {
+    code: "@supports\r\n(animation-name: test) { }",
+  }, {
+    code: "@keyframes\nidentifier { }",
+  }, {
+    code: "@keyframes\r\nidentifier { }",
+  }, {
+    code: "@keyframes\r\nidentifier\n{ }",
+  }, {
+    code: "@keyframes\r\nidentifier\r\n{ }",
+  }, {
+    code: "@viewport { }",
+  }, {
+    code: "@viewport{ }",
+  }, {
+    code: "@viewport\n{ }",
+  }, {
+    code: "@viewport\r\n{ }",
+  }, {
+    code: "@viewport\n\n{ }",
+  }, {
+    code: "@viewport\r\n\r\n{ }",
+  }, {
+    code: "@counter-style\nwinners-list { }",
+  }, {
+    code: "@counter-style\r\nwinners-list { }",
+  }, {
+    code: "@font-face { };",
+  }, {
+    code: "@unknown\n\"ident\";",
+  }, {
+    code: "@unknown\r\n\"ident\";",
+  }, {
+    code: "@unknown\nident { };",
+  }, {
+    code: "@unknown\r\nident { };",
+  }, {
+    code: "a { color: pink; @crazy-custom-at-rule; }",
+  } ],
+
+  reject: [ {
+    code: "@charset \"UTF-8\";",
+    message: messages.expectedAfter("@charset"),
+    line: 1,
+    column: 8,
+  }, {
+    code: "@charset\"UTF-8\";",
+    message: messages.expectedAfter("@charset"),
+    line: 1,
+    column: 8,
+  }, {
+    code: "@charset  \"UTF-8\";",
+    message: messages.expectedAfter("@charset"),
+    line: 1,
+    column: 8,
+  }, {
+    code: "@charset\n\n\"UTF-8\";",
+    message: messages.expectedAfter("@charset"),
+    line: 1,
+    column: 8,
+  }, {
+    code: "@charset\r\n\r\n\"UTF-8\";",
+    message: messages.expectedAfter("@charset"),
+    line: 1,
+    column: 8,
+  }, {
+    code: "@media\n\n(width <= 100px) { }",
+    message: messages.expectedAfter("@media"),
+    line: 1,
+    column: 6,
+  }, {
+    code: "@media\r\n\r\n(width <= 100px) { }",
+    message: messages.expectedAfter("@media"),
+    line: 1,
+    column: 6,
+  },  {
+    code: "@media (width <= 100px) { }",
+    message: messages.expectedAfter("@media"),
+    line: 1,
+    column: 6,
+  }, {
+    code: "@media(width <= 100px) { }",
+    message: messages.expectedAfter("@media"),
+    line: 1,
+    column: 6,
+  }, {
+    code: "@media  (width <= 100px) { }",
+    message: messages.expectedAfter("@media"),
+    line: 1,
+    column: 6,
+  }, {
+    code: "@media (min-width: 700px)\nand (orientation: landscape) { }",
+    message: messages.expectedAfter("@media"),
+    line: 1,
+    column: 6,
+  }, {
+    code: "@media (min-width: 700px)\r\nand (orientation: landscape) { }",
+    message: messages.expectedAfter("@media"),
+    line: 1,
+    column: 6,
+  }, {
+    code: "@media(min-width: 700px)\nand (orientation: landscape) { }",
+    message: messages.expectedAfter("@media"),
+    line: 1,
+    column: 6,
+  }, {
+    code: "@media(min-width: 700px)\r\nand (orientation: landscape) { }",
```

Comment:
I'm not sure this set of tests, with newlines before `and`, adds anything to the test suite. Why might this ever fail?